schema/swagger-extensions.json

```diff
@@ -1578,6 +1581,14 @@
     "xmsParameterLocation": {
       "type": "string",
       "enum": ["client", "method"]
+    },
+    "xmsMutability": {
+      "type": "array",
+      "items": {
+        "type": "string",
+        "enum": ["create, read", "update"]
+      },
+      "uniqueItems": true
```

Comment:
@tbombach - One less linter rule to assert that the extension should have unique values.